grader-host/README.md

```diff
@@ -0,0 +1,49 @@
+# Grading host(s) deployment
+
+The grading machines can be independently deployed and scaled without disturbing the web app deployment. However, there is a piece of software that has to glue both components together. This component is called the *backend*
+
+![Alt text](architecture.PNG?raw=true "Title")
+
+As you can see in the diagram, *backend* is the piece of software that makes possible the interaction between the frontend and the grading hosts.
+
+## Important
+
+Due to design decisions The backend does not need to know the hostnames of the grading hosts. However, the grading hosts DO need to know the hostname of the backend and the port that it is listening to.
+
+*That means that you need to provide in each and every grading host the hostname of the backend. Moreover, the backend should be deployed and listening BEFORE we deploy and run the grading host software*
```

Comment:
Improve this documentation line, clarify the synchronization between agents and backend